README.md

```diff
@@ -315,16 +266,260 @@ For example:
   - `fields, fields->>'c2', sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields, sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields->>'c2', sqrt((fields->>'c1')::int)`: there is no fields jsonb column, so function `sqrt()` is pushed down.
+  
+Also see [Limitations](#limitations).  
+
+Supported platforms
+-------------------
 
-## Contributing
+`influxdb_fdw` was developed on Linux, and should run on any
+reasonably POSIX-compliant system.
+
+`influxdb_fdw` is designed to be compatible with PostgreSQL 10 ~ 14.
+
+Go version should be 1.10.4 or later.
+
+Installation
+------------
+### Prerequisites
+
+Install InfluxDB Go client library
+<pre>
+go get github.com/influxdata/influxdb1-client/v2
+</pre>
+
+### Source installation
+
+Add a directory of pg_config to PATH and build and install influxdb_fdw.
+<pre>
```

Comment:
Would you please update this for me?

------

Using Go client
<pre>
make with_llvm=no GO_CLIENT=1
make install  with_llvm=no GO_CLIENT=1
</pre>

Using Influxdb_cxx client
<pre>
make with_llvm=no CXX_CLIENT=1
make install  with_llvm=no CXX_CLIENT=1
</pre>

Reply:
This is simple repeat of Your previous note, @weiting1lee.

Reply:
Yes, you're right, it's my miss.
Thanks for noticing me.